.github/check-format.sh

```diff
@@ -9,6 +9,8 @@ pipx run pre-commit run --show-diff-on-failure --color=always --from-ref "origin
 
 echo -n "Collecting information on changed files"
 
+git checkout -- dev_tools/scripts/*.sh
```

Comment:
Why is this needed ?

Reply:
It was there before. I deleted it by mistake. But it's needed to filter out formatting changes on the dev_tools scripts.